llama/patches/0020-ggml-No-alloc-mode.patch

```diff
@@ -645,16 +649,16 @@ index 02d413467..f79e5d65c 100644
 +
 +static void ggml_backend_cuda_reset(ggml_backend_t backend) {
 +    ggml_backend_cuda_context * ctx = (ggml_backend_cuda_context *)backend->context;
-+    ctx->pools[ctx->device] = NULL;
++    ctx->pools[ctx->device][ctx->curr_stream_no] = NULL;
```

Comment:
I think we need to loop over all of the streams and set them to NULL to free them. Similarly, for `ggml_backend_cuda_buffer_size()`, we also need to sum up the alloc sizes for all of the streams.